src/oc/mswinprt.c

```diff
@@ -364,7 +364,8 @@ int getpid() {
 void hoc_Plt() {ret(); pushx(0.);}
 void hoc_Setcolor(){ret(); pushx(0.);}
 void hoc_Lw(){ret(); pushx(0.);}
-void hoc_settext(){ret(); pushx(0.);}
+// TODO : just temporary fix to avoid duplicate symbol error
```

Comment:
Shouldn't this TODO be taken care of by this PR?

Reply:
@nrnhines : Is this TODO relevant here?